API/Domain/Position.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace API.Domain;
+
+public class Position
+{
+    public int Id { get; set; }
+    public string Name { get; set; } = string.Empty;
+

```

Comment:
Let's stick to the version with no blank lines between "normal" properties and ICollection